content/concepts/multiplex/yamux.md

```diff
@@ -1,15 +1,33 @@
 ---
-title: "yamux"
+title: "Yamux"
 description: "yamux is a multiplexing protocol designed by Hashicorp."
 weight: 180
 ---
 
-[yamux](https://github.com/hashicorp/yamux) is a multiplexing protocol designed by [Hashicorp](https://www.hashicorp.com/).
-yamux offers more sophisticated flow control than mplex, and can scale to thousands of multiplexed streams over a
-single connection.
+## What is Yamux?
```

Comment:
I suggest adding an introduction sentence of what purpose Yamux serves in the libp2p stack.